Picker/tests/Picker-specs.js

```diff
@@ -1,100 +1,105 @@
-import {mount} from 'enzyme';
+import '@testing-library/jest-dom';
+import {render} from '@testing-library/react';
+
 import {Picker, PickerBase} from '../Picker';
 
 describe('Picker Specs', () => {
 	test('should render selected child wrapped with <PickerItem/>', () => {
-		const picker = mount(
-			<Picker value={1}>
+		const {getByTestId} = render(
+			<Picker data-testid="picker" value={1}>
 				{[1, 2, 3, 4]}
 			</Picker>
 		);
 
+		const picker = getByTestId('picker');
 		const expected = '2';
-		const actual = picker.find('PickerItem').text();
+		const actual = picker.children.item(1).textContent;
 
 		expect(actual).toBe(expected);
 	});
 
 	test(
 		'should set the max of <Picker> to be one less than the number of children',
```

Comment:
Maybe add this on a single line like the other test descriptions.

Reply:
Ok